complex.c

```diff
@@ -2350,30 +2364,27 @@ float_arg(VALUE self)
 }
 
 /*
- * A \Complex object houses a pair of values
- * called, respectively, the _real_ and _imaginary_ parts;
- * see {Complex number}[https://en.wikipedia.org/wiki/Complex_number].
- *
- * Note that each of the parts may be a an instance of class Numeric,
- * or an instance of one of its subclasses:
- * Complex, Float, Integer, or Rational.
+ * A \Complex object houses a pair of values,
+ * given when the object is created as either <i>rectangular coordinates</i>
+ * or <i>polar coordinates</i>.
  *
- * You can create a \Complex object with:
+ * == Rectangular Coordinates
  *
- * - A {complex literal}[rdoc-ref:syntax/literals.rdoc@Complex+Literals].
- * - \Method {Kernel#Complex}[https://docs.ruby-lang.org/en/master/Kernel.html#method-i-Complex].
- * - Methods Complex.rect or Complex.polar.
- * - Methods Numeric#to_c or String#to_c;
- *   or (trivially) methods Complex#to_c or NilClass#to_c.
+ * The rectangular coordinates of a complex number
+ * are called the _real_ and _imaginary_ parts;
+ * see {Complex number definition}[https://en.wikipedia.org/wiki/Complex_number#Definition].
  *
- * == Rectangular Coordinates
+ * You can create a \Complex object from rectangular coordinates with:
  *
- * Each of the methods above (except Complex.polar) takes two "rectangular" arguments
- * representing the _real_ and _imaginary_ parts of the created \Complex object;
- * see {Complex definition}[https://en.wikipedia.org/wiki/Complex_number#Definition].
+ * - A {complex literal}[rdoc-ref:doc/syntax/literals.rdoc@Complex+Literals].
+ * - \Method Complex.rect.
+ * - \Method {Kernel#Complex}[https://docs.ruby-lang.org/en/master/Kernel.html#method-i-Complex],
```

Comment:
What version of RDoc are you on? I'm on `6.6.1` and `Kernel#Complex` links correctly for me.

Reply:
$ rdoc --version
6.3.1

Reply:
I know that Kernel#Complex links correctly on docs.ruby-lang.org, so have complied here.